cartridges/int_cloudinary/cartridge/scripts/jobs/steps/cldUploadContentLibraryAssets.js

```diff
@@ -78,7 +78,8 @@ module.exports.Start = function (args) {
                         continue;
                     }
 
-                    assetPublicID = jobStepHelpers.getAssetRelURL(args.file.toString());
+                    const file = args && args.file ? args.file.toString() : '';
```

Comment:
you can define file as
`args?.file?.toString() || '';`

Reply:
"?" doesn't work in SFCC

Reply:
thanks, good to know